chroma_core/models/host.py

```diff
@@ -1816,3 +1816,29 @@ def run(self, kwargs):
         self.invoke_rust_agent_expect_result(
             kwargs["host"], "snapshot_unmount", {"fsname": kwargs["fsname"], "name": kwargs["name"]}
         )
+
+
+class MountSnapshotJob(HostListMixin):
+    fsname = models.CharField(max_length=512)
```

Comment:
I think FS name is limited to 8 chars, so there might not be need to reserve more.

Reply:
Done